docs/html-css/week-1/lesson.md

```diff
@@ -174,19 +174,19 @@ When creating a web page, we should always think about how to structure the cont
 
 ### Sectioning Content
 
-We can group elements with 'sectioning' tags.
+We can apply structure to a web page using 'sectioning' elements.
```

Comment:
I think grouping elements is important to say clearly.  This is because I often teach catchup classes where trainees get a lot of value from discussing and reflecting on the concept of naming and grouping data in html. This is all structuring a page, of course, but the grouping word is useful. 

How about going right to the spec?
```suggestion
We can group elements with sectioning content.  This splits a page up into named sections.
```

Reply:
```suggestion
We can put groups of content that make up areas of a web page inside 'sectioning' elements. These split the page up into named sections that help to organise the page's structure.
```

I've tried to reflect the way [the part of the HTML spec dealing with sections](https://html.spec.whatwg.org/multipage/sections.html) refers to grouping of content (I'm hoping this is exactly what you're referring to by "grouping elements"). I'm not sure the use of "sectioning content" works here - it has a [particular meaning](https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2) in the spec that does not include the `main`, `header` and `footer` elements mentioned here in the diagram (though does include `aside` and other sub-section groupings like `article` and `section`), and I'm not sure using "sectioning content" in the more vernacular way is clear either.

What do you think @SallyMcGrath ? I do think we're close to something that is hopefully clear and that includes that sense of grouping HTML data within these more macrostructural elements.